tests/ai_ml/tflite2.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: This module install tensorflow2-lite and downloads 
+#   a test program in python, a model, labels and an image.
+#   Then, it runs the model with the image as input.
+# Maintainer: Guillaume GARDET <guillaume@opensuse.org>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use Utils::Architectures 'is_aarch64';
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_serial_terminal;
+    # Install required software
+    zypper_call('in tensorflow2-lite python3-Pillow');
+
+    select_console('user-console');
+    # Perform tests in a separate folder
+    script_run('mkdir tflite2_tests && pushd tflite2_tests');
+
+    # Download python test program
+    assert_script_run('wget https://github.com/tensorflow/tensorflow/raw/master/tensorflow/lite/examples/python/label_image.py');
```

Comment:
This file can change at any time. Wouldn't it make more sense to pin it to a specific version? And besides, wouldn't it make more sense to have these files within the os-autoinst-distri-opensuse repository instead of downloading them over and over again and relying on a network connection and GitHubs availability. The same counts for the other downloads.

Other tests also store files and even images within the tests repository, e.g. there's `data/imagemagick`. Note that I'm mainly involved with developing openQA itself (instead of maintaining tests) so let's here what the test maintainers have to say about that.

Reply:
On it.